kivy/tests/test_window_base.py

```diff
@@ -18,3 +18,26 @@ def test_to_normalized_pos(self):
                 assert result_sy == expected_sy
         finally:
             win.system_size = old_system_size
+
+
+class WindowOpacityTest(GraphicUnitTest):
+
+    def test_window_opacity_property(self):
```

Comment:
This method can be moved into `WindowBaseTest` as adding new class `WindowOpacityTest` doesn't seem necessary for just one method.